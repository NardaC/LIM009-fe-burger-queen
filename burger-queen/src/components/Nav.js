import React from "react";

const Nav = ({ logo }) => (
  <header>
    <img data-testid="imgLogo" className="center pb-1 pt-1" src={logo} alt="logo"></img>
  </header>
);
export default Nav;
