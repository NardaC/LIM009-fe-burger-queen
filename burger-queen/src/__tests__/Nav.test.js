import React from 'react';
import { render, fireEvent, getByAltText } from '@testing-library/react';
import '@testing-library/jest-dom';
import '@testing-library/react/cleanup-after-each'
import '@testing-library/jest-dom/extend-expect'
import Nav from '../components/Nav';
import  Logo from '../img/logoBq.png';

it('renders', () => {
    const { asFragment } = render( <Nav logo ={Logo} />);
    expect(asFragment()).toMatchSnapshot();
})
it('inserts logo image in <img>', () => {
    const{ getByTestId, getByAltText} =render (<Nav logo={Logo}/>)
    expect(getByTestId('imgLogo')).toHaveClass('center');
    expect(getByAltText('logo')).toHaveAttribute('alt', expect.stringContaining('logo'));
})
