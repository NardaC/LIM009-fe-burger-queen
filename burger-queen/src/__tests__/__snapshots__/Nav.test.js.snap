// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <header>
    <img
      alt="logo"
      class="center pb-1 pt-1"
      data-testid="imgLogo"
      src="logoBq.png"
    />
  </header>
</DocumentFragment>
`;
